packages/graphql/tests/e2e/setup/ws-client.ts

```diff
@@ -42,11 +47,30 @@ export class WebSocketTestClient {
     }
 
     public waitForNextEvent(): Promise<void> {
-        if (this.onEvent) {
-            return Promise.reject(new Error("Cannot wait for multiple events"));
-        }
-        return new Promise<void>((resolve) => {
-            this.onEvent = resolve;
+        return this.waitForEvents(1);
+    }
+
```

Comment:
I see that we are using `waitForEvents(1)` in our tests instead of this method so maybe we could remove this altogether